pages/index.tsx

```diff
@@ -1,82 +1,154 @@
+  content: string;
+  data: {
+    title: string;
+    image: string;
+    url: string;
+    subtitle: string;
+  };
+};
 
-export default function IndexPage() {
-  const imageUrl = "https://ep2023.europython.eu/social-cards/default.png";
-  const metaRef = useRef<HTMLMetaElement>(null);
+type Intro = {
+  source: any;
+  data: {
+    title: string;
+  };
+};
 
-  useEffect(() => {
-    console.log("metaRef", metaRef.current);
-    if (metaRef.current) {
-      // add comment before meta tag
-      metaRef.current.before(document.createComment("GNU Terry Pratchett"));
-    }
-  }, []);
+type Venue = {
+  source: any;
+  data: {
+    title: string;
+    map: string;
+  };
+};
 
+export default function IndexPage({
+  deadlines,
+  intro,
+  venue,
+}: {
+  deadlines: Deadline[];
+  intro: Intro;
+  venue: Venue;
+}) {
   return (
-    <>
-      <Head>
-        <meta charSet="utf-8" />
-        <meta
-          name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no"
-        />
-
-        <meta httpEquiv="cleartype" content="on" />
-        <meta httpEquiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-
-        <link
-          rel="alternate"
-          type="application/rss+xml"
-          title="EuroPython Blog RSS Feed"
-          href="http://blog.europython.eu/rss"
-        />
-
-        <title>
-          EuroPython 2023 | July 17th-23rd 2023 | Prague, Czech Republic &
-          Remote
-        </title>
-        <meta
-          name="title"
-          content="EuroPython 2023 | July 17th-23rd 2023 | Prague, Czech Republic & Remote"
-        />
-        <meta name="description" content="" />
-        <meta name="author" content="EuroPython" />
-
-        <link rel="icon" href="/epslogo.png" type="image/png" />
-
-        <meta property="og:type" content="website" />
-        <meta property="og:url" content="https://ep2023.europython.eu" />
-        <meta
-          property="og:title"
-          content="EuroPython 2023 | July 17th-23rd 2023 | Prague, Czech Republic & Remote"
-        />
-        <meta property="og:description" content="" />
-        <meta property="og:image" content={imageUrl} />
-
-        <meta property="twitter:card" content="summary_large_image" />
-        <meta property="twitter:url" content="https://ep2023.europython.eu" />
-        <meta
-          property="twitter:title"
-          content="EuroPython 2023 | July 17th-23rd 2023 | Prague, Czech Republic & Remote"
-        />
-        <meta property="twitter:description" content="" />
-        <meta property="twitter:image" content={imageUrl} />
-
-        <Clacks />
-      </Head>
-
-      <div className="flex items-center justify-center content-center min-h-screen flex-col">
+    <Layout>
+      <Fullbleed>
         <Hero />
+      </Fullbleed>
+
+      <CardContainer>
+        {deadlines.map(({ data, content }) => (
+          <Card
+            key={data.title}
+            image={data.image}
+            url={data.url}
```

Comment:
I don't know how we render to the front end but can we make sure that people can see what is a URL and what isn't, it took me a while to figure out that the headings were actually a URL.

Reply:
You look at the preview: https://ep2023-git-home-page-europython-society.vercel.app/

Reply:
@patrick91 I personally find the CTA boxes to be clearly clickable. But I am not sure if that's the case for everyone. Since it was raised, could you double check?

Reply:
On mobile, there's no real indication that you can click on these three cards. With a mouse, you get a pointing hand cursor on hoover, but there's obviously no such indication on mobile.